pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -692,12 +710,20 @@ public void close() throws IOException {
         messagePublishBufferMonitor.shutdown();
         consumedLedgersMonitor.shutdown();
         backlogQuotaChecker.shutdown();
-        authenticationService.close();
+        try {
+            authenticationService.close();
+        } catch (IOException e) {
+            return FutureUtil.failedFuture(e);
```

Comment:
Should we have a try/catch for the entries method instead of the individual checks. That would also ensure we don't forget any of them (and we also handle unchecked exceptions)

Reply:
done.